src/Makefile.am

```diff
@@ -193,6 +193,7 @@ BITCOIN_CORE_H = \
   rpc/request.h \
   rpc/server.h \
   rpc/util.h \
+  saltedhash.h \
```

Comment:
Maybe place these in `util` or ``crypto? I'd prefer not to add files to the source root anymore when possible.

Reply:
Moved to util.